src/apps/activity_assignments/domain/assignments.py

```diff
@@ -79,3 +80,15 @@ def validate_assignments(cls, values):
 
 class ActivitiesAssignmentsDelete(InternalModel):
     assignments: list[ActivityAssignmentDelete]
+
+
+class AssignmentsSubjectCounters(PublicModel):
+    respondents: set[uuid.UUID] = Field(default_factory=set)
+    subjects: set[uuid.UUID] = Field(default_factory=set)
+    subject_assignments_count: int = 0
+    respondent_assignments_count: int = 0
+
+
+class AssignmentsActivityCountBySubject(PublicModel):
```

Comment:
Same observation here…
```suggestion
class AssignmentsActivityCountBySubject(InternalModel):
```